graphql-server/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.18 as builder
```

Comment:
let's specify to use bullseye, in case of base image change on their side, the built binary might not be portable to another os